src/components/[guild]/Requirements/components/GuildCheckout/components/FeeAndTotal.tsx

```diff
@@ -0,0 +1,139 @@
+import {
+  Divider,
+  HStack,
+  Icon,
+  Skeleton,
+  Stack,
+  Text,
+  Tooltip,
+  useColorModeValue,
+} from "@chakra-ui/react"
+import { formatUnits } from "@ethersproject/units"
+import { RPC } from "connectors"
+import useTokenData from "hooks/useTokenData"
+import { Info } from "phosphor-react"
+import {
+  GUILD_FEE_FIXED_USD,
+  GUILD_FEE_PERCENTAGE,
+} from "utils/guildCheckout/constants"
+import usePrice from "../hooks/usePrice"
+import usePurchaseAsset from "../hooks/usePurchaseAsset"
+import { useGuildCheckoutContext } from "./GuildCheckoutContex"
+
+const FeeAndTotal = (): JSX.Element => {
+  const { pickedCurrency, requirement } = useGuildCheckoutContext()
+
+  const textColor = useColorModeValue("gray.800", "gray.200")
+  const textAccentColor = useColorModeValue("black", "white")
```

Comment:
Isn't `colorScheme="gray"` on the Text components and no colorScheme where textAccentColor is used sufficient?